app/views/shared/_links.haml

```diff
@@ -1,5 +1,6 @@
 %li= link_to 'diasporafoundation.org', "https://diasporafoundation.org"
 %li= link_to 'Wiki', "https://wiki.diasporafoundation.org"
+%li= link_to 'Forum', "https://discourse.diasporafoundation.org"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.